recipes/fastmcp/recipe.yaml

```diff
@@ -0,0 +1,63 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: fastmcp
+  version: "2.3.4"
+  python_min: '3.10'
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/fastmcp-${{ version }}.tar.gz
+  sha256: f3fe004b8735b365a65ec2547eeb47db8352d5613697254854bc7c9c3c360eea
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  python:
+    entry_points:
+      - fastmcp = fastmcp.cli:app
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - hatchling
+    - uv-dynamic-versioning >=0.7.0
+    - pip
+  run:
+    - python >=${{ python_min }},<4
+    - python-dotenv >=1.1.0
+    - exceptiongroup >=1.2.2
+    - httpx >=0.28.1
+    - mcp >=1.8.1,<2.0.0
+    - openapi-pydantic >=0.5.1
+    - rich >=13.9.4
+    - typer >=0.15.2
+    - websockets >=14.0
+    - authlib >=1.5.2
+
+tests:
+  - python:
+      imports:
+        - fastmcp
+      pip_check: true
+      python_version: ${{ python_min }}.*
+  - requirements:
+      run:
+        - pip
```

Comment:
I don't think you need `pip` here. But I think it should be replaced with `- python ${{ python_min }}.*`